test/python/primitives/test_backend_sampler.py

```diff
@@ -275,6 +275,7 @@ def test_primitive_job_status_done(self, backend):
         bell = self._circuit[1]
         sampler = BackendSampler(backend=backend)
         job = sampler.run(circuits=[bell])
+        job.result()
```

Comment:
It might be good to clarify that you intentionally ignore the result though it's not a strong opinion.
```suggestion
        _ = job.result()
```

Reply:
Thank you. I like this suggestion.